console/backend/src/main/java/org/frankframework/management/web/Init.java

```diff
@@ -41,25 +38,27 @@
 @RestController
 public class Init extends FrankApiBase {
 
-	@Autowired
-	protected @Getter HttpServletRequest servletRequest;
+	private final RequestMappingHandlerMapping handlerMapping;
 
-	@Autowired
-	private RequestMappingHandlerMapping handlerMapping;
+	public Init(RequestMappingHandlerMapping handlerMapping) {
+		this.handlerMapping = handlerMapping;
+	}
```

Comment:
?

Reply:
Constructor based injection ipv deprecated autowired

Reply:
Ja duidelijk. Ook beter dat hij dan nu final is.